### ISSUE TITLE
I2C

### ISSUE THREAD

**Issue by AlvaroGordo** [Roboteurs/slushengine#16]
Hey, I’m thinking about buy Pi Hat module of Slushengine, I want to communicate raspberry with arduino through I2C pins on UEXT. Do you know is it possible?

**Comment by legel** [comment]
“Hey, I'm thinking about buy Pi Hat module of Slushengine, I want to communicate raspberry with arduino between I2C pin on UEXT. Do you know is it possible?”

// I think the answer is almost certainly *yes*

My colleague in Amsterdam, Andrejs, may be able to help confirm at some point: andrejs@3co.ai

Cheers,
Lance

**Comment by Mango-kid** [comment]
Hey Alvaro,

Yes the I2C pins are routed straight from the pi to the UEXT header. Pinout
is the same as specified by Olimex.

**Comment by AlvaroGordo** [comment]
Yes, I think so about UEXT conection, but I don’t find Pi HAT schematic, and I think I’ve seen Raspberry need to enable I2C and SPI interface as pre-requisites to Slush library works.

**Comment by Mango-kid** [comment]
Please see attaced.

Yes you will need to enabled these interfaces. It can be done via
"raspi-config"

**Comment by AlvaroGordo** [comment]
Yeah, but is used I2C interface by Raspberry to communicate with the Pi HAT? That's my question.

**Comment by Mango-kid** [comment]
Ah. No the Slushengine uses the SPI bus to communicate with the Raspberry
Pi.

**Comment by vanvught** [comment]
> No the Slushengine uses the SPI bus to communicate with the Raspberry

And also I2C is used for the MCP23017 and the temperature sensor (MAX1164). The latter never worked for me.
